src/pages/Success/ThankYou/index.js

```diff
@@ -3,33 +3,28 @@ import React, {useContext} from 'react';
 // import context
 import AppContext from '../../../context/AppContext';
 
+import Yes from './Yes';
+import No from './No';
+
```

Comment:
perhaps we should use a more descriptive name like withName and withoutName or something, what do you think?

Reply:
`Yes` is because they selected yes to Giftaid which is the default for Submit and an option on Update. `No` only applies for updates

Reply:
oh right sorry, didn't read that properly

Reply:
I renamed them to`GiftAidConfirmMessage` and `GiftAidCancelMessage` to be more descriptive. What do you think?